js/modules/keyboard-navigation.src.js

```diff
@@ -86,6 +86,29 @@ function stripTags(s) {
 }
 
 
+/**
+ * Get the index of a point in a series. This is needed when using e.g. data
+ * grouping.
+ *
+ * @param {Point} point The point to find index of.
+ * @return {number} The index in the series.points array of the point.
+ */
+function getPointIndex(point) {
+    var index = point.index,
+        points = point.series.points,
+        i = points.length;
+    if (points[index] !== point) {
```

Comment:
Just out of curiousity, in what cases could the `point.index` be wrong?

Reply:
`point.index` basically points into `xData`/`yData`, not `series.points`. `series.points` only includes the visible points: https://jsfiddle.net/07hc8jts/